configure.ac

```diff
@@ -63,6 +63,10 @@ AC_SUBST([gdkmm_CFLAGS_SYSTEM], ['$(subst -I,-isystem ,$(gdkmm_CFLAGS))'])
 AC_SUBST(gerbv_LIBS)
 AC_SUBST(gerbv_CFLAGS)
 
+# GERBV_INTERPOLATION_LINEARx10 isn't available in older gerbv.h's, but
+# we want to use the new symbol
+AX_C_COMPILE_VALUE(GERBV_INTERPOLATION_LINEARx10,NEW_LINEARx10,gerbv.h)
```

Comment:
Should be like this:

```
AX_C_COMPILE_VALUE(GERBV_INTERPOLATION_LINEARx10,LINEARx10,[#include "gerbv.h"])
```

But now it's complaining that it can't find gerbv.h .  I tried first this:

```
CFLAGS+="$(gerbv_CFLAGS)"
```

But it didn't work in configure.ac.  I'm not sure how to get at gerbv_CFLAGS from within configure.ac.  Can it be done?

Reply:
Ah, this seems to work:

```
CFLAGS+=" ${gerbv_CFLAGS}"
AX_C_COMPILE_VALUE(GERBV_INTERPOLATION_LINEARx10,LINEARx10,[#include "gerbv.h"])
```

In config.log, there are no errors and it seems to figure out that the value of that variable is 0x0001, which is correct.

Reply:
Want to try that, see if it works for you?  If so, and it's working for me, I'll merge.

Reply:
Yup, that builds for me.

FWIW, this is what it finds for the variable when GERBV_INTERPOLATION_LINEARx10 is missing:

  checking value of GERBV_INTERPOLATION_LINEARx10 by analyzing object code... 0

So it looks like it's working correctly for you as well.